clients/tests/test_cr.cpp

```diff
@@ -27,25 +27,37 @@
 #include <gtest/gtest.h>
 #include <vector>
 
-typedef std::tuple<int, std::string, unsigned int> cr_tuple;
+typedef std::tuple<int, std::string, unsigned int, int> cr_tuple;
 
 std::vector<int>         cr_size = {7, 63};
 std::vector<std::string> cr_precond
     = {"None", "Chebyshev", "FSAI", "Jacobi", "SGS", "ILU", "ItILU0", "IC", "MCSGS"};
-std::vector<unsigned int> cr_format = {2, 4, 7};
+std::vector<unsigned int> cr_format  = {2, 4, 7};
```

Comment:
const?

Reply:
We can't here as we `push_back` and `insert` to it.